package.json

```diff
@@ -13,6 +13,7 @@
     "compressible": "~2.0.18",
     "debug": "2.6.9",
     "negotiator": "~0.6.4",
+    "object-assign": "~4.1.1",
```

Comment:
To me, the inclusion of this library alone justifies major version rev'ing. How do other folks feel about this? We obviously could publish this on the current line and then follow with a major so that folks on the current lines can use it, but it would be nice to keep things moving toward dropping old node support while we work on these.

Reply:
I think this is up to the maintainer, and they’ve expressed interest in making it available for the current major release. Although I generally agree, I’d like to allow the maintainer to decide.

Reply:
I don’t think it’s best to remove support for older versions and not be able to provide the feature that has been awaited for years in this package in the current major version.

Reply:
Ok, I will not fight it and I agree it is nice to add in the existing major range.